.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+# Description
+
+Include a summary of the change and what was done. Also include relevant motivation and context for the change. If any, list dependencies are required for this change.
+
+Include the Asana link, or relevant issue. (https://app.asana.com/0/187002107997407/list)
+
+Don't forget to add reviewers on github and mention the PR in the appropriate discord channel.
+
+## Testing
+
+Please describe the tests that you ran to verify your changes. Provide instructions and details so the changes can be tested if required. Also mention who performed the testing.
+
+## Documentation
+
+Include links to changes in documentation on confluence that you have created, if it was required.
+
+https://uwarg-docs.atlassian.net/wiki/spaces/ZP/overview?homepageId=20217925
+
+# Merge Checklist:
+
+- [ ] The changes have been well commented, particularly in hard-to-understand areas.
+- [ ] The code has been tested on hardware, either by me or somone else.
+- [ ] Comprehensive unit tests have been made for this change
+- [ ] Corresponding changes to documentation have been created and links to this documentation are provided within the pull request.
+- [ ] The changes generate no new warnings and compile and run; A screenshot of a successful compile message is attached to the PR.
```

Comment:
Where would this be attached? I am not sure where I would put the image based on this

Reply:
Any suggestions on a preferred location? Maybe just `... A screenshot of a successful compile message is attached to the bottom of this PR.`